architecture/ecr/update_tiktok/update.tf

```diff
@@ -14,7 +14,7 @@ provider "aws" {
 }
 
 resource "aws_ecr_repository" "long_term_image" {
-  name                 = "spotify-tiktok-1-hour-update"
+  name                 = "spotify-tiktok-update-tiktok"
```

Comment:
Maybe a better variable name for the ecr could make it more clear what the image is doing